### DISCUSSION TITLE
Make "Skip" Button clickable with left and right button

### DISCUSSION THREAD

**Discussion by Uniter343** [ppy/osu#20630] [Ideas] [answered]
It is currently only clickable with the left mouse button (or by pressing space). This is inconvenient for people who have a playstyle with access to neither of these (for example tablet and keypad).
In stable you could click the skip button with the normal assigned "osu!std" keys which easily solved this.

**Comment by Uniter343** [comment]
The same thing also goes for The "Continue", "Retry" and "Back" Buttons in the Pause Menu.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/3551 please search firs
